Azure/sonic-buildimage
# [dockers] label SONiC Docker with manifest

Signed-off-by: Stepan Blyschak <stepanb@nvidia.com>

This PR is part of [SONiC Application Extension](https://github.com/stepanblyschak/SONiC/blob/sonic-app-ext-3/doc/sonic-application-extention/sonic-application-extention-hld.md)

Depends on https://github.com/Azure/sonic-buildimage/pull/5938



**- Why I did it**
To provide an infrastructure change in order to support SONiC Application Extension feature.

**- How I did it**
Label every installable SONiC Docker with a minimal required manifest and auto-generate packages.json file based on
installed SONiC images.

**- How to verify it**
Build an image, execute the following command:

```
admin@sonic:~$ docker inspect docker-snmp:1.0.0 | jq '.[0].Config.Labels["com.azure.sonic.manifest"]' -r | jq
```

Cat /var/lib/sonic-package-manager/packages.json file to verify all dockers are listed there.

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
